testing/xcb-util-xrm/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
+pkgname=xcb-util-xrm
+pkgver=1.2
+pkgrel=0
+pkgdesc="Utility functions for the X resource manager"
+url="https://github.com/Airblader/xcb-util-xrm/"
+arch="all"
+license="MIT"
+makedepends="libx11-dev libxcb-dev xcb-util-dev util-macros bsd-compat-headers autoconf automake libtool"
+source="https://github.com/Airblader/xcb-util-xrm/releases/download/v1.2/xcb-util-xrm-$pkgver.tar.gz"
+subpackages="$pkgname-dev"
+
+builddir="$srcdir/$pkgname-$pkgver"
+build() {
+	cd "$builddir"
+	autoreconf -fi || return 1
```

Comment:
Do you have some special reason why you run `autoreconf`? The source tarball already contains `./configure` script.